spring-cloud-kubernetes-client-config/src/test/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientSecretsPropertySourceLocatorTests.java

```diff
@@ -112,10 +113,10 @@ void getLocateWithSources() {
 		sources.add(source1);
 		sources.add(source2);
 		secretsConfigProperties.setName("app");
-		secretsConfigProperties.setNamespace("");
```

Comment:
actually it is rather good that after I made my changes in this PR, these tests started failing. It proves that a namespace must be provided, otherwise we will fail